lib/fungus_toast/games/games.ex

```diff
@@ -81,53 +87,60 @@ defmodule FungusToast.Games do
       {:error, %Ecto.Changeset{}}
 
   """
+  #TODO Dave says there may be some opportunities here... need to run all updates in a transaction, and pulling values from the attrs might be odd
   def create_game(attrs \\ %{}) do
     changeset = %Game{} |> Game.changeset(attrs)
     user_name = Map.get(attrs, :user_name) || Map.get(attrs, "user_name")
 
     with {:ok, game} <- create_game_for_user(changeset, user_name) do
+      game	
+        |> Players.create_ai_players()
+
+      grid_size = Map.get(attrs, :grid_size)
```

Comment:
@jakejgordon I pulled this down and ran tests. Looks like `grid_size` is `nil` here and we aren't providing a default.
That said, we do create the game before calling this and `game` has `game.grid_size` set at this point (defaulted if none is provided). We should remove this all-together and just get it off game IMO.

Reply:
@jakejgordon my last commit should address this. There are still some tests that need love but this helps with some of the failures 👍